.gitignore

```diff
@@ -34,3 +34,7 @@ product-test-reports
 **/dependency-reduced-pom.xml
 core/trino-web-ui/src/main/resources/webapp/dist/
 .node
+
+# Local plugins added for testing should be excluded
+testing/trino-server-dev/etc/plugin/*
```

Comment:
Isn't this redundant with `testing/trino-server-dev/etc/plugin/.gitignore`?

Reply:
Yes! This was added before I added the nested `.gitignore`. Thanks for catching it.